plugins/pluginmanagers/lazy.nix

```diff
@@ -0,0 +1,216 @@
+
+              build =
+                helpers.mkNullOrOption
+                (oneOf [
+                  (either str (listOf str))
+                  helpers.nixvimTypes.rawLua
+                ])
+                "build is executed when a plugin is installed or updated. Before running build, a plugin is first loaded. If it's a string it will be ran as a shell command. When prefixed with : it is a Neovim command. You can also specify a list to executed multiple build commands. Some plugins provide their own build.lua which is automatically used by lazy. So no need to specify a build step for those plugins.";
+
+              submodules = helpers.defaultNullOpts.mkBool true "When false, git submodules will not be fetched. Defaults to true";
+
+              event =
+                helpers.mkNullOrOption
+                (oneOf [
+                  (either str (listOf str))
+                  helpers.nixvimTypes.rawLua
+                ])
+                "Lazy-load on event. Events can be specified as BufEnter or with a pattern like BufEnter *.lua";
+
+              cmd =
+                helpers.mkNullOrOption
+                (oneOf [
+                  (either str (listOf str))
+                  helpers.nixvimTypes.rawLua
+                ])
+                "Lazy-load on command";
+
+              ft =
+                helpers.mkNullOrOption
+                (oneOf [
+                  (either str (listOf str))
+                  helpers.nixvimTypes.rawLua
+                ])
+                "Lazy-load on filetype";
+
+              keys =
+                helpers.mkNullOrOption
+                (oneOf [
+                  (either str (listOf str))
+                  helpers.nixvimTypes.rawLua
+                ])
+                "Lazy-load on key mapping";
+
+              module = helpers.defaultNullOpts.mkBool false "Do not automatically load this Lua module when it's required somewhere";
+
+              priority = helpers.defaultNullOpts.mkNum null "Only useful for start plugins (lazy=false) to force loading certain plugins first. Default priority is 50. It's recommended to set this to a high number for colorschemes.";
+
+              optional = helpers.defaultNullOpts.mkBool false "When a spec is tagged optional, it will only be included in the final spec, when the same plugin has been specified at least once somewhere else without optional. This is mainly useful for Neovim distros, to allow setting options on plugins that may/may not be part of the user's plugins";
+
+              opts = helpers.mkNullOrOption (either attrs helpers.nixvimTypes.rawLua) "opts should be a table (will be merged with parent specs), return a table (replaces parent specs) or should change a table. The table will be passed to the Plugin.config() function. Setting this value will imply Plugin.config()";
```

Comment:
helpers.mabyeRaw (attrsOf str)

Please avoir using `attrs`, and instead use `attrsOf str`, merging is handling better.

Reply:
The ops is a table, so (attrsOf str) will fail.
`error: A definition for option `plugins.lazy.plugins."[definition 1-entry 7]".opts.ensure_installed' is not of type `string'. Definition values:
       - In `<unknown-file>': { }`

Reply:
Ah sorry, attrsOf anything